.drone.yml

```diff
@@ -0,0 +1,12 @@
+image: bradrydzewski/base
+cache:
+  - /var/run
```

Comment:
This is a HACK to work around the lack of direct Docker volumes support. This allows the container to use the host's Docker.